backend/src/main/java/de/neuefische/backend/dto/CovidTestDto.java

```diff
@@ -0,0 +1,29 @@
+package de.neuefische.backend.dto;
+
+import de.neuefische.backend.model.Result;
+import de.neuefische.backend.model.TestType;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+import java.time.LocalDateTime;
+
+
+@AllArgsConstructor
+@NoArgsConstructor
+@Data
+@Builder
+public class CovidTestDto {
```

Comment:
This class could probably be implemented as a record (exists since Java 14). It would look like this:
```java
public record CovidTestDto(String id, TestType testType, LocalDateTime dateTime, Result result) {
}
```
Records are imutable, automatically generate hasCode, equals, toString and getters (only that the get methods don't have get in the name. So it's id() not getId()).